src/components/sessions/SessionTimeline.tsx

```diff
@@ -132,12 +143,24 @@ function SessionTimeline({
 
   const isFeedbackSession = blocks.length === 0
 
+  const [runtime, setRuntime] = useState(calculateRuntime({ startedAt }))
+
+  const startingTime = runtime.includes('d')
+    ? dayjs(startedAt).format('DD.MM. HH:mm:ss')
+    : dayjs(startedAt).format('HH:mm:ss')
+
+  useEffect(() => {
```

Comment:
Should add a cleanup handler as well (i.e., return a function that will be called when the component is unmounted).

E.g., ```
var handle = setInterval(...)
return () => clearInterval(handle)
```

Reply:
Thanks for reviewing, implemented it as suggested.